pages/dkp/kommander/1.4/clusters/attach-cluster/index.md

```diff
@@ -2,32 +2,36 @@
 layout: layout.pug
 beta: true
 navigationTitle: Attach Kubernetes Cluster
-title: Attach Cluster
+title: Attach Kubernetes Cluster
 menuWeight: 7
-excerpt: A guide for attaching an existing Kubernetes cluster using kubeconfig
+excerpt: A guide for attaching an existing Kubernetes cluster
 ---
+<!-- markdownlint-disable MD034 -->
+## Using Kommander with an existing Kubernetes cluster
 
-## Attach Kubernetes Cluster
+You can attach an existing cluster directly to Kommander. At the time of attachment, certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure the attached cluster has sufficient resources before you begin. For more information on platform services and customizing them, see [workspace platform services][workspace_platform_services].
 
-You can attach an existing cluster directly to Kommander. At the time of attachment, certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure the attached cluster has sufficient resources. For more information on platform services and customizing them, see [workspace platform services][workspace_platform_services].
+If the cluster you want to attach was created using Amazon EKS, Azure AKS, or Google GKE, create a service account as described below. If you are attaching an Amazon EKS cluster to Kommander, [use these detailed instructions][attach_eks_cluster].
 
-If the cluster you want to attach was created using Amazon EKS, Azure AKS, or Google GKE, create a service account as described below. If you are attaching an Amazon EKS cluster to Kommander, [detailed instructions are available][attach_eks_cluster].
+### Before you begin
 
-### Before you begin (optional)
+You must have a kubeconfig file to attach a cluster. The kubeconfig file contains YAML manifest that establishes the connection between Kommander and an existing cluster. If you already have a kubeconfig file, skip this procedure and go to [Attaching a cluster](#attaching-a-cluster).
```

Comment:
Let's note that the kubeconfig file does not establish any connection but provides the required context and authentication tokens to allow Kommander to manage an existing cluster.

Reply:
Implemented as suggested.